src/tests/Dialog.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent } from '@testing-library/react'
+import { Dialog } from '../components/Dialog/Dialog.tsx'
+import { ReactNode } from 'react'
+
+jest.mock('focus-trap-react', () => {
+    return {
+        __esModule: true,
+        default: ({ children }: { children: ReactNode }) => children,
+    }
+})
+
+describe('Dialog', () => {
+    function setup() {
+        return {
+            title: 'Test Dialog',
+            children: <div>Test Children</div>,
+            onClose: jest.fn(),
+        }
+    }
+
+    it('renders without crashing', () => {
+        const props = setup()
+        const { container } = render(
+            <Dialog {...props}>
+                <div></div>
+            </Dialog>,
+        )
+        expect(container).toBeInTheDocument()
+    })
+
+    it('renders the correct title', () => {
+        const props = setup()
+        const { getByText } = render(
+            <Dialog {...props}>{props.children}</Dialog>,
```

Comment:
If you are not doing anything special with the children, there is no need to extract them and put explicitly between parent's tags. So this can be rewritten as `<Dialog {...props} />` without changing the meaning.